.buildkite/pipelines/emergency_release.yml

```diff
@@ -1,3 +1,20 @@
+env:
+  VERSION: ${BUILDKITE_COMMIT:0:12}
+  DOCKER_IMAGE_TAG: git-${VERSION}
```

Comment:
Are these two `DOCKER_*` env variables used?

Reply:
I pushed https://github.com/elastic/kibana/pull/176258/commits/c8a77cd9470d2ba895bf9f1a146ad29f2e8b12d5 and verified they aren't needed in the gpctl-promote pipeline.